cmd/cafe.go

```diff
@@ -37,18 +37,25 @@ Use this command to add, list, get, remove cafes and check messages.
 
 type addCafesCmd struct {
 	Client ClientOptions `group:"Client Options"`
+	Token  string        `required:"true" short:"t" long:"token" description:"An access token supplied by the Cafe."`
 }
 
 func (x *addCafesCmd) Usage() string {
 	return `
 
-Registers with a cafe and saves an expiring service session token.`
+Registers with a cafe and saves an expiring service session token.
+Unless explicitly disabled by the Cafe, an access token is required to register.
```

Comment:
This isn't actually currently possible, but wanted to have it here for discussion. Any reason to allow a Cafe to disable tokens? Basically leave itself wide open for registrations?

Reply:
sounds like an enhancement

Reply:
Yup, agreed, removing text.